matlab/lib/+Ice/CloseConnectionException.m

```diff
@@ -7,24 +7,6 @@
 % propagated to the application code.
 
 % Copyright (c) ZeroC, Inc. All rights reserved.
-% Generated from LocalException.ice by slice2matlab version 3.7.10
 
-classdef CloseConnectionException < Ice.ProtocolException
-    methods
-        function obj = CloseConnectionException(ice_exid, ice_exmsg, reason)
-            if nargin <= 2
-                reason = '';
-            end
-            if nargin == 0 || isempty(ice_exid)
-                ice_exid = 'Ice:CloseConnectionException';
-            end
-            if nargin < 2 || isempty(ice_exmsg)
-                ice_exmsg = 'Ice.CloseConnectionException';
-            end
-            obj = obj@Ice.ProtocolException(ice_exid, ice_exmsg, reason);
-        end
-        function id = ice_id(~)
-            id = '::Ice::CloseConnectionException';
-        end
-    end
+classdef (Sealed) CloseConnectionException < Ice.ProtocolException
```

Comment:
A typical exception that is thrown only from the C++ code. It inherits its ctor from ProtocolException, which itself inherits its ctor from LocalException.